packages/ui-kit/src/components/rss-feeds.js

```diff
@@ -1,121 +1,94 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
-import css from '@emotion/css';
 import useSWR from 'swr';
 import Parser from 'rss-parser';
-import moment from 'moment';
+import PropTypes from 'prop-types';
 import LoadingSpinner from '@commercetools-uikit/loading-spinner';
 import ContentNotifications from './content-notifications';
-import { colors, tokens, dimensions } from '../design-system';
-import Link from './link';
-
-const Table = styled.table`
-  width: 100%;
-  border-radius: ${tokens.borderRadiusForRssFeedTable};
-  box-shadow: ${tokens.shadowForRssFeedTable};
+import RssFeedTable from './rss-feed-table';
 
-  thead {
-    th {
-      background-color: ${colors.light.surfaceSecondary2};
-      border-radius: ${tokens.borderRadiusForRssFeedTable}
-        ${tokens.borderRadiusForRssFeedTable} 0 0;
-      padding: ${dimensions.spacings.s};
-    }
-  }
+async function fetcher(...args) {
+  const rssParser = new Parser();
+  const promises = args.map(async (url) => {
+    const feedData = await rssParser.parseURL(url);
+    const feedName = feedData.title.split(' ').slice(1, 3).join(' ');
```

Comment:
```suggestion
    const feedName = feedData.title.replace(/^commercetools (.*) Release Notes$/, "$1");
```

<img width="449" alt="image" src="https://user-images.githubusercontent.com/1110551/98942863-efdd3480-24ee-11eb-8cf5-58b87e1cf14f.png">

Reply:
👍